File: python/prograimg_python_basic/Exercises/conditional_statements/conditional_statements_advanced_-_exercise/new_house.py
type_flower = str(input())
count_flower = int(input())
budget = int(input())

discount = 0

roses = 5
dahlias = 3.80
tulips = 2.80
narcissus = 3
gladiolus = 2.50

end_sum =0

if type_flower == "Roses":
    sum_for_flower = count_flower * roses
    if count_flower > 80:
        discount = 0.10
    sum_discount = (sum_for_flower * discount)
    end_sum = (sum_for_flower - sum_discount)

elif type_flower == "Dahlias":
    sum_for_flower = count_flower * dahlias
    if count_flower > 90:
        discount = 0.15
    sum_discount = (sum_for_flower * discount)
    end_sum = (sum_for_flower - sum_discount)

elif type_flower == "Tulips":
    sum_for_flower = count_flower * tulips
    if count_flower > 80:
        discount = 0.15
    sum_discount = (sum_for_flower * discount)
    end_sum = (sum_for_flower - sum_discount)

elif type_flower == "Narcissus":
    sum_for_flower = count_flower * narcissus
    if count_flower < 120:
        discount = 0.15
    sum_discount = (sum_for_flower * discount)
    end_sum = (sum_for_flower + sum_discount)

elif type_flower == "Gladiolus":
    sum_for_flower = count_flower * gladiolus
    if count_flower < 80:
        discount = 0.20
    sum_discount = (sum_for_flower * discount)
    end_sum = (sum_for_flower + sum_discount)


if budget >= end_sum:
    left = budget - end_sum
    print(f"Hey, you have a great garden with {count_flower} {type_flower} and {'{:.2f}'.format(left)} leva left.")

elif budget < end_sum:
    need_more = end_sum - budget
    print(f"Not enough money, you need {'{:.2f}'.format(need_more)} leva more.")

File: JS/JavaScript/JS-Basics/ConditionalStatementsAdvancedExercise/NewHouse.js
function house(inputs) {

    let type = String(inputs[0]);
    let count = Number(inputs[1]);
    let budget = Number(inputs[2]);

    let amount;

    if (type == "Roses") {
        amount = count * 5;

        if (count > 80) {
            amount -= amount * 0.10;
        };   
    }
    else if (type == "Dahlias") {
        amount = count * 3.80;

        if (count > 90) {
            amount -= amount * 0.15;
        };
    }
    else if (type == "Tulips") {
        amount = count * 2.80;

        if (count > 80) {
            amount -= amount * 0.15;
        };
    }
    else if (type == "Narcissus") {
        amount = count * 3;

        if (count < 120) {
            amount += amount * 0.15;
        };
    }
    else if (type == "Gladiolus") {
        amount = count * 2.50;

        if (count < 80) {
            amount += amount * 0.20;
        };

    };

    if (amount <= budget) {
        console.log(`Hey, you have a great garden with ${count} ${type} and ${(budget - amount).toFixed(2)} leva left.`);
    }
    else {
        console.log(`Not enough money, you need ${(amount - budget).toFixed(2)} leva more.`);
    };

}

house(["Roses","55","250"])
house(["Tulips","88","260"])
house(['Gladiolus', '64', '160'])